package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "13.5.0",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-icons": "^4.4.0",
```

Comment:
Remove `^`. We update manually.

Reply:
We want to have the frontend server running on `8080`, and the backend server on `3000` so we want to inject `PORT=8080` in the start script right before `react-scripts start`.